oap-server/server-receiver-plugin/skywalking-ebpf-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/ebpf/provider/handler/EBPFProfilingServiceHandler.java

```diff
@@ -100,6 +102,35 @@ public void queryTasks(EBPFProfilingTaskQuery request, StreamObserver<Commands>
         responseObserver.onCompleted();
     }
 
+    private List<EBPFProfilingTaskCommand> buildProfilingCommands(EBPFProfilingTask task, List<Process> processes) {
+        final ArrayList<EBPFProfilingTaskCommand> commands = new ArrayList<>(processes.size());
+        for (Process process : processes) {
+            // The service id must matches between process and task
+            if (!Objects.equals(process.getServiceId(), task.getServiceId())) {
+                continue;
+            }
+
+            // If the task haven't labeled requirement, them just add the process
+            boolean shouldAdd = false;
+            if (CollectionUtils.isEmpty(task.getProcessLabels())) {
+                shouldAdd = true;
+            } else {
+                // If the process have matched one of the task label, them need to add the command
```

Comment:
I think need to match all labels, not one of them.

Reply:
Do we have a UI preview to show how many processes matched? In most label system, multiple labels are all required, rather than one of, right?

Reply:
Understand. For now, we haven't a UI to show how many processes matched. But I think matching all labels is more make sense.

Reply:
Sure, please make sure this is considered as a part of the visualization design. After all, listing all is working, but no one is really going to read them one by one.

Reply:
Updated. Add the `etimateProcessScale` for it. Update the GraphQL and E2E for testing it.